src/unleash.ts

```
@@ -9,8 +10,12 @@ import { EventEmitter } from 'events';
 const BACKUP_PATH: string = tmpdir();
 
 export interface UnleashConfig {
+    appName?: string,
```

should it be `?` if you throw if it's missing?